### ISSUE TITLE
"Get out of blueprint's way" constructor pathing issue

### ISSUE THREAD

**Issue by sprEEEzy** [beyond-all-reason/Beyond-All-Reason#3463]
### Description

Con building an Air lab.

https://github.com/user-attachments/assets/3d12017a-9918-4e27-bb1d-9a2be993709e

### Expected Behaviour

Moves top left.

### Actual Behaviour

Moves all the way down, because top side is blocked by con turrets?

### Reproduction steps

Cannot consistently reproduce. Encountered another bug which I'll present in another ticket.

**Comment by drivver44** [comment]
please raise on engine repo

**Comment by sprEEEzy** [comment]
Closing, made new on engine repo. https://github.com/beyond-all-reason/spring/issues/1618
